transcom/mymove
# B 20069 int change labels and add 110% prime estimated weight value

## [B-20069](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A950768)

## Summary

This story renames “Weight requested” to "Actual weight" and "Total billable weight" to "Actual billable weight". This also will display the weight for 110% of the prime estimated weight for HHG and NTS shipments. All changes are on the Payment Request page when signed in as a TIO.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login as a Customer
2. Create an HHG and an NTS shipment and submit them
3. Login as a Service Counselor and approve the shipments
4. Login as a TOO and approve the shipments
5. Login as the Prime Simulator access the shipment
6. Click update shipment for HHG and enter pickup dates and delivery dates in the future
7. Enter weights for Estimated and Actual weights (remember your estimated weights or what 110% of it should be)
8. Fill out rest of fields and save
9. Repeat above for the NTS shipment
10. Login as a TIO, you will likely have to do a search for your move code
11. Go to the Payment Request page
12. Verify the Actual weight label is present instead of Weight requested
13. Verify Actual billable weight label is there instead of Total billable weight
14. Underneath the HHG and NTS shipments you should now see 110% of the estimated weight you entered as a Prime
![image](https://github.com/transcom/mymove/assets/149000222/403a59f6-ebd9-42a8-b567-8d10062e59dd)

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:
